src/graph.c

```diff
@@ -33,7 +37,7 @@ int RAI_GraphInit(RedisModuleCtx* ctx) {
       .digest = NULL
   };
 
-  RedisAI_GraphType = RedisModule_CreateDataType(ctx, "DL__GRAPH", 0, &tmGraph);
+  RedisAI_GraphType = RedisModule_CreateDataType(ctx, "AI__GRAPH", 0, &tmGraph);
```

Comment:
if you change the datatype name it will not be backward compatible, is it ok?

Reply:
I don't think we have actual users outside a tiny group. Also, backward compatibility here applies to databases being restored from RDB or AOF right? In that case don't save to those yet.